.github/workflows/docs-check.yml

```diff
@@ -38,9 +40,6 @@ jobs:
 
       - uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265 # v19.1.0
         with:
-          globs: |
-            Documentation/**/*.md
-            !Documentation/Helm-Charts
```

Comment:
Markdown lint is now being run twice. Once above, and once here.

Reply:
right. An oversight. fixed.

Reply:
This is outdated as the changes to the workflow have been removed from the PR.